copilot/kubernetes-argocd-passthru.py

```
@@ -0,0 +1,103 @@
+
+# ArgoCD Configuration
+ARGOCD_SERVER = "https://argocd.yourcluster.com"
+ARGOCD_TOKEN = "your-argocd-token"  # Consider using secure secret management
+
+@app.api_route("/k8s/{full_path:path}", methods=["GET", "POST", "PUT", "DELETE"])
+async def kubernetes_passthru(full_path: str, request: Request):
+    """
+    Pass-through endpoint for Kubernetes API
+    Allows direct proxying of Kubernetes API requests
+    """
+    k8s_client = get_k8s_client()
```

Ruff (0.8.2)

Local variable `k8s_client` is assigned to but never used

Remove assignment to unused variable `k8s_client`

(F841)